### PR TITLE
feat: Create journal filter

### PR DESCRIPTION
**Pull request by IscoRuta**
# Pull Request
Create endpoints that:
1. Get list of journals
2. Get all articles of a specific journal
## Scope
https://dev.azure.com/CPerrie/Aarons%20Kit%20-%20Information%20Storage/_workitems/edit/52
## Work Done

- Created a journal.py file
- In the journal.py file, created two functions that cater to points 1. and 2.
- Updates the rest.py file
## Steps to Test
1. Open the branch on your local text editor.
2. Run `docker compose up`
3. In a web browser, type: `http://localhost:5000/api/journals/titles`  to get list of all journals.
4. In a web browser, type: `http://localhost:5000/api/journals?journalName=Economic Theory
`  to get a list of all articles in a particular journa
articles_of_specific_journal
l
journal_list
.

### CODE REVIEW

**Comment by CailinPerrie** [comment]
Looks great Fransisco.

File: app/journal.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    cursor.execute(query)
+    results = cursor.fetchall()
+
+    resp = jsonify(results)
+
+    resp.status_code = 200
+
+    return resp
+
+#Get all articles of a specific journal
```
Comment (DominicGBauer) [inline app/journal.py]: delete this comment the function name explains what this does

File: app/journal.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import pymysql
+from app import app
+from db import mysql
+from flask import jsonify, request
+from flask_cors import cross_origin
+
+# Get list of journals
```
Comment (DominicGBauer) [inline app/journal.py]: delete function name explains this

File: app/journal.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+#Get all articles of a specific journal
+@app.route("/api/journals")
+@cross_origin()
+def getSpecificArticlesfromJournals():
+    query_parameters = request.args
+    journal = query_parameters.get('journalName')
+
+    query = """
+    SELECT Title, URL, DOI, YearPublished, JournalName, CategoryName, AuthorInitial, AuthorSurname
```
Comment (DominicGBauer) [inline app/journal.py]: can you make this SQL query neater please, follow what I did with a more complex SQL query on system-design-backend

**Review by DominicGBauer — CHANGES_REQUESTED** [review]

File: app/journal.py (hunk 3 above)
Comment (IscoRuta) [reply app/journal.py]: I have applied the changes you requested.

**Review by DominicGBauer — APPROVED** [review]

**Pull request merged**